mopidy/core/actor.py

```diff
@@ -136,6 +138,106 @@ def tags_changed(self, tags):
             self.playback._stream_title = title
             CoreListener.send('stream_title_changed', title=title)
 
+    def on_start(self):
```

Comment:
That makes sense. 
Back to the 'names' issue: Have you a proposal for the function names: 
`init` and `deinit`
`setup` and `shutdown`